models/user.go

```diff
@@ -296,9 +296,6 @@ func (u *User) GenerateRandomAvatar() error {
 	if err != nil {
 		return fmt.Errorf("RandomImage: %v", err)
 	}
-	// NOTICE for random avatar, it still uses id as avatar name, but custom avatar use md5
-	// since random image is not a user's photo, there is no security for enumable
-	u.Avatar = fmt.Sprintf("%d", u.ID)
```

Comment:
This line will fix when u.Avatar is empty, then line 305 `fw, err := os.Create(u.CustomAvatarPath())` will create an avatars file. This bug is reported by #1114.